Tasks/DownloadBuildArtifacts/task.json

```diff
@@ -6,10 +6,9 @@
   "helpMarkDown": "",
   "category": "Utility",
   "author": "Microsoft Corporation",
-  "preview": true,
   "version": {
     "Major": 0,
-    "Minor": 131,
+    "Minor": 132,
```

Comment:
need to port change to release branch.

when porting, keep Minor as 131 & bump up patch? so do the same change here as well?